### PR TITLE
xyz color space

### PR DESCRIPTION
**Pull request by estelle**
clarification as to what the channels are (an impossible task, but giving people the gist.

### CODE REVIEW

File: files/en-us/glossary/color_space/index.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ CIELab color spaces, including Lab, Lch, Oklab, and Oklch, are device-independen
...
 
-People with normal vision have three kinds of cone cells that sense light, having peaks of spectral sensitivity in differing wavelengths. The CIE X, Y, and Z parameters correspond to levels of stimuli of the three kinds of cone cells which, in principle, describe every color. The CIE 1931 XYZ color space, with its tri-stimulus values, encompasses all visible colors.
+People with normal vision have three kinds of cone cells that sense light, having peaks of spectral sensitivity in differing wavelengths. The CIE X, Y, and Z parameters correspond to levels of stimuli of the three kinds of cone cells which, in principle, describe every visible color. The `Y` channel represents the luminance of a color. The `Z` channel reflect the amount of blue in the color, but is not the same as the `B` in RGB. The `X` axis is orthogonal to the Y- and Z- axis the XYZ color 3-D coordinate system.
```
Comment (chrisdavidmills) [inline files/en-us/glossary/color_space/index.md:87]:
```suggestion
People with normal vision have three kinds of cone cells that sense light, having peaks of spectral sensitivity in differing wavelengths. The CIE X, Y, and Z parameters correspond to levels of stimuli of the three kinds of cone cells which, in principle, describe every visible color. The `Y` channel represents the luminance of a color. The `Z` channel reflects the amount of blue in the color but is not the same as the `B` in RGB. The `X` axis is orthogonal to the Y- and Z- axis of the XYZ color 3D coordinate system.
```

Comment (chrisdavidmills) [reply files/en-us/glossary/color_space/index.md:87]: But even then, this doesn't really make sense. When you say "The `X` axis is orthogonal to the Y- and Z- axis the XYZ color 3-D coordinate system", what do you mean? We are currently describing XYZ, so how can X be orthogonal to Y and Z in the same system?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@estelle noble attempt to try and clarify this, but I'm finding the new wording a bit confusing, TBH.

File: files/en-us/glossary/color_space/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/glossary/color_space/index.md:87]: it's a 3d space, so think "cube". y and z make a flat plane, and x is at a right angle to that.

**Review by chrisdavidmills — APPROVED** [review]
@estelle, ah, I get this now. Thanks for the clarification.

**Pull request merged**